Client/Frontend/Sync/SyncViewController.swift

```diff
@@ -12,6 +12,13 @@ class RoundInterfaceButton: UIButton {
     }
 }
 
+class RoundInterfaceView: UIView {
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        layer.cornerRadius = bounds.height / 2.0
```

Comment:
Will this be okay rounding based only on the height or should it be `min(width, height) / 2.0` or `max(width, height) / 2.0`

Reply:
It would be `min` I believe. This was ported from a button, which generally isn't portrait, but good point. I will adjust.